dev-libs/xxhash/xxhash-0.6.4.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A fast non-cryptographic hash algorithm"
+DESCRIPTION="xxHash is an extremely fast non-cryptographic hash algorithm, \
+working at speeds close to RAM limits."
+HOMEPAGE="https://cyan4973.github.io/xxHash/
+	http://www.xxhash.com/"
+COPYRIGHT="2012-2016 Yann Collet"
```

Comment:
A few cosmetic changes in the `COPYRIGHT` years:
* s/2012/2014/ (I'm not sure about this, but the development seems to have started only in July 2014, as we can see that there has been [not a single commit until July 2014](https://github.com/Cyan4973/xxHash/graphs/contributors?from=2012-04-29&to=2014-07-01&type=c))
* s/2016/2017/ ([v0.6.4 was released on December 22, 2017](https://github.com/Cyan4973/xxHash/releases/tag/v0.6.4))

Reply:
Fixed.